src/txmempool.h

```diff
@@ -187,11 +280,69 @@ struct TxMempoolInfo
  *
  * CTxMemPool::mapTx, and CTxMemPoolEntry bookkeeping:
  *
- * mapTx is a boost::multi_index that sorts the mempool on 3 criteria:
+ * mapTx is a boost::multi_index that sorts the mempool on 2 criteria:
```

Comment:
The change to 2 here is confusing, given that there are 3 bullet points below.

Reply:
There was noise here due to differeing things we have / haven't backported. In the interest of avoiding another CI run I'm gonna consider this a typo to be fixed later.